<launch>
    <!--
    Debug file that uses image_pub node.
    -->
    <arg name="pub_rate" default="1" />

    <node pkg="stereo_dnn_ros_viz" type="stereo_dnn_ros_viz_node" name="stereo_dnn_ros_viz" output="screen">
        <param name="camera_topic_left"  value="/zed/left/image_rect_color" />
        <param name="camera_topic_right" value="/zed/right/image_rect_color" />
        <param name="dnn_topic"          value="/stereo_dnn_ros/network/output" />
        <param name="viz_topic"          value="/stereo_dnn_ros_viz/output" />
        <param name="in_queue_size"      value="10" />
    </node>

</launch>
